packages/checkbox/src/CheckboxPresenter/CheckboxPresenter.scss

```diff
@@ -0,0 +1,158 @@
+// @import '~@hig/styles/mixins/states.scss';
+@import '../../../../node_modules/@hig/styles/mixins/colors';
+@import '../../../../node_modules/@hig/styles/tokens/layers';
+
+.hig__input-button {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0 10px;
+}
+
+.hig__input-button__inputV1 {
+  opacity: 0;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: $checkbox-layer;
+}
+
+.hig__input-button__label {
+  font-size: 14px;
+  font-weight: normal;
+  font-family: 'ArtifaktElement', sans-serif;
+  color: color(hig-cool-gray-50);
+
+  margin: 8px;
+  line-height: 14px;
+
+  .hig__input-button--required &::after {
+    content: '*';
+  }
+  .hig__input-button--checked & {
+  }
+}
+
+.hig__input-button__inputV1:checked[disabled], .hig__input-button__inputV1[disabled]  ~ .hig__input-button__label {
+  color: color(hig-cool-gray-30);
+}
+
+
+.hig__input-button__input-wrapper {
+  position: relative;
+  color: transparent;
+  border: 1px solid color(hig-cool-gray-30);
+  background-color: transparent;
+  text-align: center;
+  font-size: 25px;
+  line-height: 14px;
+  height: 16px;
+  width: 16px;
+  border-radius: 3px;
+
+  .check--indeterminate,
+  .check--disabled,
+  .check,
+  &.checked.disabled .check,
+  &.indeterminate .check,
+  &.indeterminate .check--disabled,
+  &.indeterminate.checked .check,
+  &.indeterminate.checked .check--disabled,
+  &.indeterminate.checked.disabled .check,
+  &.indeterminate.checked.disabled .check--disabled {
```

Comment:
Since these classes aren't using BEM, it's really difficult understand what's going on here. Regardless, there shouldn't be a case where we have to nest styles under a chain of classes like this. This is a code smell pointing to a lack of encapsulation in the components.

`.check` has enough presentational logic to be its own component. I suggest creating a `CheckPresenter` component, and moving all of it's related styles into it's own respective stylesheet.

Reply:
Yeah, this was a good suggestion.  We've taken a stab at encapsulating it that way in d3ee1434f63d66d3b84bac371ec1ff9cf0a70bf0.  Let us know your thoughts and if it's worthwhile to do gemini/storybook tests for the `CheckPresenter`

Will follow up with BEM next